packages/frontend-v2/pages/home.tsx

```diff
@@ -251,6 +261,7 @@ const HomePage: NextPage = () => {
             coReqErr={undefined}
             preReqErr={undefined}
             isRemove={isRemove}
+            isFromSidebar={activeCourse.id.startsWith(SIDEBAR_DND_ID_PREFIX)}
```

Comment:
this is important so that we don't style courses from the sidebar with the "delete course" styles